glfs.c

```diff
@@ -563,16 +563,10 @@ static int tcmu_glfs_open(struct tcmu_device *dev, bool reopen)
 		if (round_down(st.st_size, block_size) == dev_size)
 			goto out;
 
-		if (!reopen) {
-			ret = -EINVAL;
-			goto close;
-		}
+		if (!reopen)
+		    tcmu_dev_warn(dev, "unmatched sizes in none resize path!\n");
```

Comment:
I think you wanted "non resize path". Maybe a better warning would be:

Device size mismatch during initialization.

Or, you probably do not need the extra log statement. You could just modify the one below.

Reply:
Yeah, will fix the warning.
This case is rare, normally it shouldn't be here when initializing the device, so give a warning will be more friendly.
Thanks